pkg/feature/domain/flag_trigger.go

```diff
@@ -88,3 +90,22 @@ func (ft *FlagTrigger) UpdateTriggerUsage() error {
 	ft.TriggerCount = ft.TriggerCount + 1
 	return nil
 }
+
+func (ft *FlagTrigger) GenerateToken() error {
+	secret := NewFlagTriggerSecret(
```

Comment:
Because we now save the token, we no longer need to save the UUID in the DB. Also, we just need the UUID (One-time) to generate the token.

https://github.com/bucketeer-io/bucketeer/pull/624/files#diff-5559c5f4ce4b05e5cce1d1577c268d4987c108d1a6153348c8aa8ca76f377ed2R40-R43

Reply:
So we can remove the `uuid` column for `flag_trigger` table.

Reply:
Yes.

Reply:
I looked at your last commit removing the column from MySQL, but I couldn't find the changes removing it from the code.